lib/si-layer-cache/benches/insert_speed.rs

```diff
@@ -10,12 +11,21 @@ const ASCII_LOWER: [u8; 26] = [
 
 const ONE_MB: usize = 1_000_000;
 
+type BenchmarkCache = Cache<[u8; 1], Vec<u8>>;
+
 pub async fn fresh_cache_count(objects: &[Vec<u8>], count: usize) {
-    let tempdir = tempfile::TempDir::new_in("/home/adam/benches").expect("cannotc reate tempdir");
-    let layer_cache = LayerCache::new(tempdir).expect("cannot create layer cache");
+    let target_dir = "/tmp";
```

Comment:
Feels like we should instead allow `TempDir` to handle the parent since it has OS-specific logic to do so?

Reply:
yes, you're right!